common/unsafe/src/main/java/org/apache/spark/sql/catalyst/util/CollationSupport.java

```diff
@@ -295,17 +299,15 @@ public static UTF8String execICU(final UTF8String v, final int collationId) {
 
   public static class FindInSet {
     public static int exec(final UTF8String word, final UTF8String set, final int collationId) {
```

Comment:
a comment would be useful for developers, explaining that the trimming sensitivity is taken into account "under the hood" because the collation-aware `FindInSet` implementation relies on the collator's comparator function